panel/lxqtpanel.cpp

```diff
@@ -374,7 +372,7 @@ int LXQtPanel::getReserveDimension()
     return mHidable ? PANEL_HIDE_SIZE : qMax(PANEL_MINIMUM_SIZE, mPanelSize);
 }
 
-void LXQtPanel::setPanelGeometry(bool animate)
+void LXQtPanel::setPanelGeometry(bool animate, bool firstTime)
```

Comment:
OK, I did some tests. Removal of that "flash" is possible but, apparently, it doesn't worth the effort for two reasons: (1) Too many lines would be added to the code; and (2) the point was having a "nice" start-up but I admit I was following my personal taste with obsession.

Therefore, I'll implement your suggestion without obsession ;) Thanks for the discussion!